tango.py

```diff
@@ -71,30 +67,29 @@ def __init__(self):
             from vmms.distDocker import DistDocker
             vmms = DistDocker()
 
-        self.vmms = {Config.VMMS_NAME: vmms}
-        self.preallocator = Preallocator(self.vmms)
+        self.preallocator = Preallocator({Config.VMMS_NAME: vmms})
```

Comment:
we can do that, but it seems unnecessary because we can access it via preallocator as `self.preallocator.vmms`.